index.html

```diff
@@ -27,7 +27,7 @@
       <div class="slides">
         <section id="qrcode" data-menu-title="QR code">
           <div class="qrcode" id="qrcode-talk"/>
-          <p><a href="https://example.com/link/to/this/talk" target="_blank"
+          <p><a style="white-space: nowrap" href="https://example.com/link/to/this/talk" target="_blank"
```

Comment:
Could you please put that into the SCSS file, rather than using an inline `style` attribute?